.github/workflows/publish.yml

```
@@ -1,9 +1,10 @@
 name: ocelot.social publish CI
 
-on:
-  push:
-    branches:
-      - master
+# on:
+#   push:
+#     branches:
+#       - master
+on: [push] # for testing while developing
```

```suggestion
on:
  push:
    branches:
      - master
      - 4451-new-deployment-with-base-and-code
```
This makes it obvious that you're testing Github Actions. Another solution: https://github.com/nektos/act

---

Good suggestion @roschaefer 😍
If done …